css/styles.css

```diff
@@ -66,20 +75,21 @@
 } */
 
 .row-2{
-    width: 100%;
+    width: 95%;
```

Comment:
it is fine to manipulate width here due to the fact that you made this custom class yourself. but before doing so, try again to see if the `.row` will do the job on its own first.